packages/cozy-harvest-lib/src/components/KonnectorConfiguration/Success/DriveLink.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { ButtonLink } from 'cozy-ui/react/Button'
+import { queryConnect } from 'cozy-client'
+
+import withLocales from '../../hoc/withLocales'
+
+export class DriveLink extends PureComponent {
+  render() {
+    const { folderId, driveQuery, t } = this.props
+    let hasDrive = false
+    if (driveQuery.fetchStatus === 'loaded') {
+      if (driveQuery.data.length > 0) {
+        hasDrive = true
+      }
+    }
+    const href =
+      hasDrive && `${driveQuery.data[0].links.related}#/files/${folderId}`
```

Comment:
```suggestion
      hasDrive ? `${driveQuery.data[0].links.related}#/files/${folderId}` : null
```

This has the benefit of removing the condition in ButtonLink prop.